tt_metal/llrt/blackhole/bh_hal_active_eth.cpp

```diff
@@ -74,6 +74,8 @@ HalCoreInfoType create_active_eth_mem_map() {
             .fw_base_addr = eth_l1_mem::address_map::FIRMWARE_BASE,
             .local_init_addr = eth_l1_mem::address_map::FIRMWARE_BASE,  // this will be uplifted in subsequent commits
                                                                         // enabling active erisc
+            .fw_launch_addr = 0xFFB14008,
+            .fw_launch_addr_value = (uint32_t)eth_l1_mem::address_map::FIRMWARE_BASE,
```

Comment:
minor: wasn't clear what "fw_launch_addr_value" was in other contexts, maybe "fw_launch_jmp_insn" (yes, ties the use to the name, but I think that's ok)

Reply:
hm the reason I didn't add `jump` into the name is because we don't always write the jump instruction at this value. like this case above we just write FW base addr directly to the PC